### PR TITLE
Feat(NIPT): Mark samples as controls

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
### Added

- NIPT: Mark samples as controls

### How to prepare for test
- [x] ssh to clinical-db
- [x] paxa the environment (`us; paxa --server clinical-db --resource cgweb-stage`)
- [x] install on stage:
`cd servers/resources/clinical-db.sclifelab.se; bash update-cgweb-stage.sh [THIS-BRANCH-NAME]`
- [x] ssh to clinical-db server 
- [x] paxa the environment: `paxa`
- [x] install on stage :
`bash servers/resources/clinical-db.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

**How to test ordering a non-control, a positive and a negative control from json **:
- [x] login on https://clinical-stage.scilifelab.se/
- [x] import an fluffy order-json where the first three samples are marked as non-control, a positive and a negative controls

**Expected test outcome**:
- [x] check that the samples were imported as non-control, a positive and a negative control

**How to test ordering a non-control, a positive and a negative control**:
- [x] login on https://clinical-stage.scilifelab.se/
- [x] do create an fluffy order
- [x] mark first three samples as non-control, a positive and a negative control
- [x] submit

**Expected test outcome**:
- [x] check that the samples were saved as non-control, a positive and a negative control
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by BS
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Document in changelog
- [ ] Deploy this branch
- [ ] Inform to Jesper, MR

### CODE REVIEW

File: cg/meta/orders/status.py [hunk 1]
```diff
@@ -521,17 +522,18 @@ def store_rml(
...
                 order=order,
                 ordered=ordered,
                 ticket=ticket,
-                application_version=application_version,
             )
             for sample in pool["samples"]:
                 new_sample = self.status.add_sample(
                     application_version=application_version,
                     comment=sample["comment"],
+                    control=sample.get("control") or None,
```
Comment (barrystokman) [inline cg/meta/orders/status.py:536]:
```suggestion
                    control=sample.get("control"),
```

**Review by barrystokman — APPROVED** [review]
👍🏻

**Review by Mropat — APPROVED** [review]

**Comment by patrikgrenfeldt** [comment]
Deployed on clinical-db:
```
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 22.20.1
    Uninstalling cg-22.20.1:
      Successfully uninstalled cg-22.20.1
Successfully installed cg-22.21.0
```

Deployed on Hasta:
```
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 22.20.0
    Uninstalling cg-22.20.0:
      Successfully uninstalled cg-22.20.0
Successfully installed cg-22.21.0
INFO  [alembic.runtime.migration] Running upgrade 1c27462b49f6 -> c081458ff180, Add control to sample
```

**Comment by patrikgrenfeldt** [comment]
@Mropat , @J35P312 Orderportal supporting setting sample as control has been deployed!

**Pull request merged**